src/Compiler/TypedTree/TypedTree.fsi

```diff
@@ -2317,9 +2317,6 @@ type NonLocalEntityRef =
     /// Get the mangled name of the last item in the path of the nonlocal reference.
     member LastItemMangledName: string
 
-    /// Get the details of the module or namespace fragment for the entity referred to by this non-local reference.
```

Comment:
I don't think this is used, I'm waiting for CI to agree with me.

Reply:
Looks like it does!